kv/kv.go

```diff
@@ -85,6 +102,16 @@ type Transaction interface {
 	LockKeys(keys ...Key) error
 }
 
+// MvccSnapshot is used to get/seek a specific verion in a snaphot.
+type MvccSnapshot interface {
+	// MvccGet returns the specific version of given key, if the version doesn't
+	// exists, returns the nearest(lower) version's data.
+	MvccGet(k Key, ver Version) ([]byte, error)
+	// MvccIterator seeks to the key in the specific version's snapshot, if the
+	// version doesn't exists, return the nearest(lower) version's snaphot.
+	NewMvccIterator(k Key, ver Version) Iterator
```

Comment:
There is a release interface in Snapshot. Does MvccSnapshot need release interface?